patch/0153-platform-mellanox-Add-config3-register-support.patch

```diff
@@ -0,0 +1,56 @@
+From bf9905c1e297307b86d87a809bd42f343bd20cf6 Mon Sep 17 00:00:00 2001
+From: Oleksandr Shamray <oleksandrs@nvidia.com>
+Date: Mon, 20 Jun 2022 16:02:14 +0300
+Subject: [PATCH] platform: mellanox: Add config3 register support
+
+Add config3 register which is using to get COMEX CPU board version
+
```

Comment:
I do not see this change [in Linus’ master branch][1]. Please mention the upstream status.

[1]: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/drivers/platform/x86/mlx-platform.c?id=4616e54795cc8183ecf2a6cef6c7a9091cb1ae56